testgrid/config.yaml

```diff
@@ -5506,6 +5510,10 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gci-gce-statefulset
     base_options: 'include-filter-by-regex=sig-scheduling'
     description: 'scheduling gce statefulset e2e tests for master branch'
+  - name: pod-preemption
+    test_group_name: ci-kubernetes-e2e-gce-pod-preemption
```

Comment:
Does this look reasonable @krzyzacy?

Reply:
nit: gce-pod-preemption (in case you add a job for a different provider)